### PR TITLE
setprop #2

### PR DESCRIPTION
**Pull request by gorodmi**
If your pull request is **not** translation or serverlist-related, read the list of requirements below and check each box:

- [x] I have read the [contribution guidelines](https://github.com/Anuken/Mindustry/blob/master/CONTRIBUTING.md).
- [x] I have ensured that my code compiles, if applicable.
- [x] I have ensured that any new features in this PR function correctly in-game, if applicable.

less cursed than the previous one (no block pos changing, no block size changes)
java_jmvaoz0UtA java_NWjuAD9qIv java_HaQPruALTZ
java_kHNNgTHEnF

reactorHeat - can only be changed for reactors
storedPower - can only be changed for blocks with buffered power
payload - sets payload for payload conveyors, adds payload for units

basically sets properties that are defined in LProperty enum or items / liquids
would be very cool for world processor maps

uhhh

### CODE REVIEW

**Comment by MEEPofFaith** [comment]
image

Flags is in Block, not BuildingComp
~~Unless your `flag` isn't referring to this flag.~~

**Comment by gorodmi** [comment]
not this flag, unit flag
~~actually forgot about this~~

**Comment by Mnemotechnician** [comment]
(you should reformat your code to make it follow anuke's convention)

**Comment by hhhi17** [comment]
This is a very cool idea. I hope she waits for her addition.

**Comment by BasedUser** [comment]
testing time

**Comment by BasedUser** [comment]
i've went through all the content setting and everything else, seems like this is very stable to me

things i've tried:
setting the property to null or a number
setting the target to null or an invalid object
setting the value to null or an invalid object
setting the value to extreme floating point values
setting the value to fractional floating point values

**Comment by Ruichka** [comment]
No merge 3 months later. RIP?

**Comment by Sky4ce09** [comment]
can we have this for christmas

**Comment by Xeononyum** [comment]
very cool

**Comment by TheEt1234** [comment]
can we get rereview on this

**Comment by Anuken** [comment]
This is still [extremely broken](https://discord.com/channels/391020510269669376/396416151032299521/1080307762518564884), and the videos shown are just the tip of the iceberg. Please don't try and re-submit this, I'll do it myself eventually.
